pydantic/_internal/_generate_schema.py

```diff
@@ -166,8 +164,35 @@ def modify_model_json_schema(
     return json_schema
 
 
+class Definitions:
```

Comment:
Please can we stick to a "newpaper style" in terms of order of types added to modules - public objects should come first and supporting object should be defined later in the module.

I know we're already broken this rule in this file a log, but I'd rather add this near the end.

Could we add a docstring explaining what this does?